doc/modules/ROOT/examples/ipv6.edn

```diff
@@ -0,0 +1,21 @@
+(defattrs :vertical [:plain {:writing-mode "vertical-rl"}])
+(defattrs :box-middle {:borders #{:left :right}})
+(def column-labels (mapv #(number-as-hex % 2) (range 32)))
```

Comment:
Where is this function defined? And why is it invoked using the `#()` reader macros for anonymous functions?

Reply:
Oh, I see it now. But I would not define this as a function, Clojure and Java have built in ways to do this that are simpler than what you used. Let me remind myself what they are.

Reply:
Indeed. Please don’t add `number-as-hex`, and instead everywhere you were using it, instead use:
```clojure
#(format "%02x" %)
```

Reply:
And I see I should simplify my existing code in `hex-text`, and get rid of the refer of `clojure.pprint`. Proof that as we use languages we get better at doing things simply!

Reply:
Nice, thanks for these pointers, I reused the approach you had implemented in `hex-text`.

Reply:
Yeah, I guessed! Sorry I led you to a more baroque solution than was called for.

Reply:
Hmm.. I'm not sure `format` is available in ClojureScript, it seems to have been removed from the API in [v0.0-1885](https://cljs.github.io/api/cljs.core/format). Can you confirm?

Reply:
Oh interesting, that be be the explanation for why I did it this weird way. This is the only project I use ClojureScript for (oh, I guess I do a bit in my automated blind controller project as well) so I know very little about it compared to JVM Clojure. Sorry, this just proves I should not try to make drive-by responses distractedly during the work day, and wait until I have time to pull things down and fully try them end-to-end.

Reply:
Thanks for your patience! I am leaning towards going with your function and using it in `hex-text` as well, but need to think about it while I am not also pair programming other complex tasks with my colleagues that pay the bills. 😅